www/docs/en/dev/platform_plugin_versioning_ref/index.md

```diff
@@ -23,224 +23,282 @@ description: How to manage platforms and Cordova CLI versions.
 
-
 ## Platform Versioning
 
-### Saving platforms
-To save a platform, you issue the following command :
+### Saving Platforms
+
+To save a platform, issue the following command:
 
 ```bash
-$ cordova platform add <platform[@<version>] | directory | git_url>
+cordova platform add <platform[@<version>] | directory | git_url>
 ```
 
-After running the above command, the resulting config.xml looks like :
-
-```xml
-<?xml version='1.0' encoding='utf-8'?>
-    ...
-    <engine name="android" spec="~4.0.0" />
-    ...
-</xml>
+After running the above command, the **`package.json`** should contain something as seen below:
+
+```json
+"cordova": {
+  "platforms": [
+    "android"
+  ]
+},
+"dependencies": {
+  "cordova-android": "^8.0.0",
+}
 ```
-After running the above command, the resulting package.json looks like :
 
-```bash
-"cordova": {"platforms": ["android"]},"dependencies": {"cordova-android": "^4.0.0"}
-```
-The '--nosave' flag prevents adding and deleting specified platforms from config.xml and package.json. To prevent saving a platform, you issue the following command :
+The `--nosave` flag prevents adding and deleting specified platforms to the `package.json` file. To prevent saving a platform, issue the following command:
 
 ```bash
-$ cordova platform add <platform[@<version>] | directory | git_url> --nosave
+cordova platform add <platform[@<version>] | directory | git_url> --nosave
 ```
 
-Some examples :
+Some Examples:
 
-  * **'cordova platform add android'** => retrieves the pinned version of the android platform, adds it to the project and then updates config.xml and package.json.
-  * **'cordova platform add android@3.7.0'** => retrieves the android platform, version 3.7.0 from npm, adds it to the project and then updates config.xml and package.json.
-  * **'cordova platform add https://github.com/apache/cordova-android.git'** => npm installs the specified cordova-android from the git repository, adds the android platform to the project, then updates config.xml and package.json and points its version to the specified git-url.
-  * **'cordova platform add C:/path/to/android/platform'** => retrieves the android platform from the specified directory, adds it to the project, then updates config.xml and package.json and points to the directory.
-  * **'cordova platform add android --nosave'** => retrieves the pinned version of the android platform, adds it to the project, but does not add it to config.xml or package.json.
-  * **'cordova platform remove android --nosave'** =>  removes the android platform from the project, but does not remove it from config.xml or package.json.  
+* **`cordova platform add android`** 
 
-### Mass saving platforms on an existing project
-If you have a pre-existing project and you want to save all the currently added platforms in your project, you can use :
+  Retrieves the pinned version of `cordova-android` platform from npm, adds it to the project and updates the `package.json` file.
```

Comment:
config.xml is not updated here any more at all with the platform?

Reply:
As far as how I interpreted this commit, https://github.com/apache/cordova-lib/commit/8371d7c1bca6b4ca6e64055054405edf8ccf11b6

`config.xml` should no longer be updated when a plugin or platform is added or updated. `config.xml` will only be modified to remove the plugin or platform when the remove command is executed.

Reply:
check @dpogue

Reply:
That is correct, Cordova 9 does not add any platforms or plugins to config.xml. It only updates package.json.